### PR TITLE
readme: add region endpoints

### PR DESCRIPTION
**Pull request by webern**
Add all of the regional endpoints to the readme.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by samuelkarp — APPROVED** [review]

**Comment by webern** [comment]
> webern force-pushed the webern:regions branch from 75cb4b1 to a48ef19

the

**Review by etungsten — APPROVED** [review]

**Review by jahkeup — APPROVED** [review]
Thanks! I've created #44 to track ways we can make using these endpoints more friendly.

Please hold off merging this PR until #38 is merged into `develop`.

**Pull request merged**
